expo/data.yaml

```diff
@@ -0,0 +1,3 @@
+datasets_dir: "D:/work/automl/datasets" # path to the datasets directory
```

Comment:
This is preferably a relative path, or left blank

Reply:
done